exercises/alphametics/src/lib.rs

```diff
@@ -1,5 +1,5 @@
 use std::collections::HashMap;
 
-pub fn solve(_: &str) -> Option<HashMap<char, u8>> {
-    unimplemented!()
+pub fn solve(input: &str) -> Option<HashMap<char, u8>> {
+    unimplemented!("Solve the alphametric {:?}", input)
```

Comment:
`alphametric` contradicts the spelling of the exercise, so the `r` should be removed

(I'm OK with this being `input`).